file-mon/src/main/java/com/wipro/ats/bdre/filemon/FileScan.java

```diff
@@ -30,6 +30,8 @@
 
     private static final Logger LOGGER = Logger.getLogger(FileScan.class);
 
+    private FileScan(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)